app/services/notify/notify-email.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+/**
+ * Sends an email using GOV.UK Notify
+ * @module NotifyEmailService
+ */
+
+const NotifyClient = require('notifications-node-client').NotifyClient
+
+const config = require('../../../config/notify.config.js')
+
+/**
+ * Send an email using GOV.UK Notify
+ *
+ * https://docs.notifications.service.gov.uk/node.html#send-an-email
+ *
+ * Notify has the option to provide an 'options' object. This is the object which populates templates placeholders
+ * (known as 'personalisation') and other options such as a 'reference' for identifying single unique message
+ * or a batch of messages.
+ *
+ * ```javascript
+ *   const options = {
+ *       personalisation: {
+ *         name: 'test', // matches the template placeholder {{ name }}
+ *       },
+ *       reference: 'ABC-123' // A unique identifier which identifies a single unique message or a batch of messages
+ *     }
+ * ```
+ *
+ * @param {string} templateId
+ * @param {string} emailAddress
+ * @param {object} options
+ *
+ * @returns {Promise<object>}
+ */
+async function go(templateId, emailAddress, options) {
+  const notifyClient = new NotifyClient(config.apiKey)
+
+  return _sendEmail(notifyClient, templateId, emailAddress, options)
+}
+
+async function _sendEmail(notifyClient, templateId, emailAddress, options) {
+  try {
+    const response = await notifyClient.sendEmail(templateId, emailAddress, options)
```

Comment:
Do we want to set some sort of predicate check on the options / emailAddress. 

I am thinking more for the 'sendLetter' which requires require at least 3 lines of address.

Reply:
Maybe a determine notify service with it's own validators. 

Email is an email, letter has 3 `address_lines`. 

We know the templates and the expected so it could be as granular as by template id. But we trust our upstream services have shaped the data correctly. Just might be worth considering instead of sending a bad payload to Notify.

Reply:
No need - notify will do a better job